fairseq/tasks/speech_to_speech.py

```diff
@@ -265,15 +283,15 @@ def setup_task(cls, args, **kwargs):
     def build_criterion(self, args):
         from fairseq import criterions
 
-        if len(self.multitask_tasks) > 0:
-            if self.args.target_is_code and args._name != "speech_to_unit":
-                raise ValueError(
-                    "set --criterion speech_to_unit for speech-to-unit loss with multitask"
-                )
-            elif not self.args.target_is_code and args._name != "speech_to_spectrogram":
-                raise ValueError(
-                    "set --criterion speech_to_spectrogram for speech-to-spectrogram loss with multitask"
-                )
+        # if len(self.multitask_tasks) > 0:
```

Comment:
I'm guessing it's because we have speech_to_unit_2pass and speech_to_spectrogram_2pass, two new criterion now. We can just check the prefix of `self.__name`.